tensorboard/webapp/widgets/data_table/data_table_test.ts

```diff
@@ -789,4 +798,125 @@ describe('data table', () => {
     expect(dataElements[3].nativeElement.innerText).toBe('1');
     expect(dataElements.length).toBe(4);
   });
+
+  describe('delete column button', () => {
+    it('emits removeColumn event when delete button clicked', () => {
+      const fixture = createComponent({
+        headers: [
+          {
+            type: ColumnHeaderType.VALUE,
+            name: 'value',
+            displayName: 'Value',
+            enabled: true,
+          },
+          {
+            type: ColumnHeaderType.RUN,
+            name: 'run',
+            displayName: 'Run',
+            enabled: true,
+          },
+          {
+            type: ColumnHeaderType.STEP,
+            name: 'step',
+            displayName: 'Step',
+            enabled: true,
+          },
+          {
+            type: ColumnHeaderType.RELATIVE_TIME,
+            name: 'relativeTime',
+            displayName: 'Relative',
+            enabled: true,
+          },
+        ],
+      });
+      fixture.componentInstance.hparamsEnabled = true;
+      fixture.detectChanges();
+      const headerElements = fixture.debugElement.queryAll(
+        By.css('.header > .col')
+      );
+
+      headerElements[3]
+        .queryAll(By.css('mat-icon'))[1]
```

Comment:
Can we have a more specific selection, use the delete-icon-container class in selector? Here and other newly added tests.

Reply:
Yeah that definitely makes sense since there are two icons now. I used delete-icon class to target the delete button, and I had to modify existing testing for the sorting button because the selection was targeting the wrong icon. For the sorting button, I used sorting-icon-container mat-icon to target the specific icon (see changes).